packages/mobile/src/escrow/ReclaimPaymentConfirmationScreen.tsx

```diff
@@ -127,7 +127,7 @@ class ReclaimPaymentConfirmationScreen extends React.Component<Props> {
     const feeCurrency = asyncFee.result?.currency
     const convertedAmount = divideByWei(payment.amount.valueOf())
     const userHasEnough = fee?.isLessThanOrEqualTo(
-      feeCurrency === CURRENCY_ENUM.DOLLAR ? dollarBalance : celoBalance
+      feeCurrency === Currency.Dollar ? dollarBalance : celoBalance
```

Comment:
Yes, but to do this I need a selector that I added in a future PR. I'll add this when I implement the sending of invites since I'll need to touch this code anyways.